frontend/src/scenes/dashboard/dashboards/DashboardsTable.tsx

```diff
@@ -1,50 +1,34 @@
 import { useActions, useValues } from 'kea'
```

Comment:
GitHub thinks this is a rename. Silly GitHub 🤷 

This is extracted from the moved Dashboards.tsx